delta/plugins/archive/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/archive/ArchiveDownloadSpec.scala

```diff
@@ -147,20 +146,20 @@ abstract class ArchiveDownloadSpec
     )
 
     def downloadAndExtract(value: ArchiveValue, ignoreNotFound: Boolean) = {
-      archiveDownload(value, project.ref, format, ignoreNotFound).map(sourceToMap).accepted
+      archiveDownload(value, project.ref, ignoreNotFound).map(sourceToMap).accepted
     }
 
     def failToDownload[R <: ArchiveRejection: ClassTag](value: ArchiveValue, ignoreNotFound: Boolean) = {
-      archiveDownload(value, project.ref, format, ignoreNotFound).rejectedWith[R]
+      archiveDownload(value, project.ref, ignoreNotFound).rejectedWith[R]
     }
 
     def rejectedAccess(value: ArchiveValue) = {
       archiveDownload
-        .apply(value, project.ref, format, ignoreNotFound = true)(Caller.Anonymous, global)
+        .apply(value, project.ref, ignoreNotFound = true)(Caller.Anonymous, global)
         .rejectedWith[AuthorizationFailed]
     }
 
-    s"provide a ${format.fileExtension} for both resources and files" in {
+    s"provide a ${Zip.fileExtension} for both resources and files" in {
```

Comment:
It might be nice to not use a variable here any more, I think it can cause trouble with tooling